docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -203,6 +203,23 @@ Cloud user, and manage your own version server.
 You can override this to use your own Teleport Kube Agent Updater image rather
 than a Teleport-published image.
 
+### `updater.pullCredentials`
+
+| Type     | Default value | Required? |
+|----------|---------------|-----------|
+| `string` | ""            | No        |
```

Comment:
The default value is not a valid value?
Should we default to `auto` and try to guess where the service is running?
I think we already do this in some other part of the code base

https://github.com/gravitational/teleport/blob/bf069282e65b90998f0946556cc649a2c0d096d2/lib/inventory/metadata/metadata.go#L278

Reply:
the issue is that you can run outside of aws/gcp... and still use their registries for storing image copies